components/GroupSelect.vue

```diff
@@ -11,37 +11,29 @@ select {
 <script>
 export default {
   props: {
+    value: {
```

Comment:
This is how to implement a `v-model` inside a custom component, this and doing `$emit('input', newVal)` elsewhere.